projects/plugins/jetpack/_inc/client/security/waf.jsx

```diff
@@ -305,6 +305,7 @@ export default connect( state => {
 	const sitePlan = getSitePlan( state );
 
 	return {
+		hasScan: siteHasFeature( state, 'scan' ),
 		bootstrapPath: getWafBootstrapPath( state ),
 		hasRulesAccess: getWafHasRulesAccess( state ),
```

Comment:
Changes look great! This component no longer needs `hasRulesAccess`, so we can at minimum remove it here. We should also check if the `getWafHasRulesAccess` is used anywhere else, and if not, we might follow that all the way up to removing the `has_rules_access` method itself.